CODEWALK.md

```diff
@@ -539,17 +539,9 @@ Vanilla `go test` also works, after build scripts have built and installed appro
 
 ## Dependencies
 
-Dependencies in go-filecoin are managed by [gx](https://github.com/whyrusleeping/gx), a content-addressed dependency manager. 
-You’ll notice that the hash of a dependency’s content appears in the import path. 
-Almost all runtime dependencies are managed by gx (mostly being other Protocol Labs-sponsored projects). 
+Dependencies in go-filecoin are managed as go modules, go's new dependency system. Please use at least go 1.11.
```

Comment:
There was a modules related bug fix in 1.11.4 so it might be worth requiring at least 1.11.4

Reply:
SGTM. Really, I'd like to merge https://github.com/filecoin-project/go-filecoin/pull/2308 first and then require 1.12.

Reply:
I think this has now been done. Let's just remove reference to any specific Go version here as it will rot.